### DISCUSSION TITLE
Switch off the PN532 Led

### DISCUSSION THREAD

**Discussion by NeeeeB** [adonno/tagreader#148] [Q&A] [answered]
Is it possible in any way to switch off the red PN 352 Led ?
Makes it too much visible IMHO...
I'm using your wonderful tagreader to enable/disable my home alarm.
Many thanks for this btw.

**Comment by jesserockz** [comment, marked as answer]
No, it is connected to the power and ground directly, you can draw over it with a sharpie to try to make it less visible.

**Reply by NeeeeB** [reply]
Ok thanks for your answer.

**Reply by NeeeeB** [reply]
I finally put a piece of black electrical tape on it ^^ problem solved lol
